content/departments/product-engineering/product/process/index.md

```diff
@@ -56,6 +56,20 @@ Launch tier levels (L1, L2, and L3) are also an important term to be aware of, a
 
 Product Management collaborates with a lot of different groups. Beyond our shared [work as a team value](../../../../company-info-and-process/values/index.md#work-as-a-team), there are some teams where collaboration is especially important and where roles and responsibilities can be unclear. To help clarify this, we have documented high level practices around how we work together.
 
+### Executive decisions
+
+There are some classes of questions that occasionally require an executive level in order to proceed, usually because they are part of a high level strategic decision or because they require making tradeoffs between different organizations or departments within the company. Some examples of these in the past which have gotten stuck are:
+
+- Pricing strategy
+- Upgrade strategy
+- Competitive strategy
+- Prioritization across org/department boundaries
+- Decisions when values conflict
+
+The goal is eventually for these to be documented in the handbook in more detail (where possible) so that decisions on the ground can be made without an escalation needed, but until that's the case the procedure is for the team who is needs clarity around any of these kinds of items to generate a proposal with how they would like to proceed. This can be presented directly to the executive team member(s) who makes the decision, or if that is unknown or they are not responding in a timely manner, then Christina will either make the decision herself or find the responsible exec team member and ensure it is followed up on. Be sure when making this request to ensure it is clear that you are escalating for her to help in this way (mentioning only that you are stuck, for example, is not a clear ask for help).
```

Comment:
```suggestion
The goal is eventually for these to be documented in the handbook in more detail where possible so that decisions on the ground can be made without an escalation needed, but until that's the case the procedure is for the team who is needs clarity around any of these kinds of items to generate a proposal with how they would like to proceed. This can be presented directly to the executive team member(s) who makes the decision, or if that is unknown or they are not responding in a timely manner, then Christina will either make the decision herself or find the responsible exec team member and ensure it is followed up on. Be sure when making this request to ensure it is clear that you are escalating for her to help in this way; mentioning only that you are stuck, for example, is not necessarily a clear ask for help.
```